smdebug/core/utils.py

```diff
@@ -78,6 +82,25 @@ def make_numpy_array(x):
         raise TypeError("_make_numpy_array does not support the" " type {}".format(str(type(x))))
 
 
+def get_aws_region_from_processing_job_arn(processing_job_arn):
+    region_match_regex_pattern = (
+        r"(us(-gov)?|ap|ca|cn|eu|sa)-(central|(north|south)?(east|west)?)-\d+"
+    )
+    return re.search(region_match_regex_pattern, processing_job_arn).group(0)
+
+
+def setup_profiler_report(processing_job_arn, opt_out=False):
```

Comment:
Who is going to call this function? Is PROFILER_TELEMETRY_URL a real, working URL?